packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/invalid-template-attribute/multiple-attributes/actual.html

```diff
@@ -1,6 +1,6 @@
 <template>
     <section>
-        <template if:true={showItems} for:each={items} for:item="item" onfoo={handleFoo}>
+        <template if:true={showItems} for:each={items} for:item="item" onfoo={handleFoo} class="slds-m-around_medium">
```

Comment:
Don't change existing fixtures, since it could hide regressions. It's preferable to create a brand new fixture for this.